websites/D/Discords.com/presence.ts

```diff
@@ -3,351 +3,508 @@ const presence = new Presence({
-					presenceData.buttons = [
-						{
-							label: "View Top-10",
-							url: document.location.href,
-						},
-					];
-				} else if (document.location.pathname.includes("/templates/id/")) {
-					presenceData.largeImageKey = "discordtemplates_logo";
-					presenceData.details = "Viewing Template:";
-					presenceData.state = `${
-						document.querySelector("h1.font-semibold.truncate")?.textContent ??
-						"Unknown"
-					}`;
-					presenceData.buttons = [
-						{
-							label: "View Template",
-							url: document.location.href,
-						},
-					];
-				} else if (document.location.pathname.includes("/templates/users/")) {
-					presenceData.largeImageKey = "discordtemplates_logo";
-					presenceData.details = "Viewing User:";
-					presenceData.state = `${
-						document.querySelector("h1.text-3xl.font-semibold")?.textContent ??
-						"Unknown"
-					}`;
-					presenceData.buttons = [
-						{
-							label: "View User",
-							url: document.location.href,
-						},
-					];
-				} else if (document.location.pathname.includes("/templates/search/")) {
-					presenceData.largeImageKey = "discordtemplates_logo";
-					presenceData.details = "Searching for:";
-					presenceData.state = `${
-						document.location.pathname.split("/templates/search/")[1] ??
-						"Unknown"
-					}`;
-					presenceData.smallImageKey = "search";
-					presenceData.smallImageText = "Searching...";
-				} else if (document.location.pathname.includes("/templates/tags/")) {
-					presenceData.largeImageKey = "discordtemplates_logo";
-					presenceData.details = "Searching by tag:";
-					presenceData.state = `${
-						document.location.pathname.split("/templates/tags/")[1] ?? "Unknown"
-					}`;
-					presenceData.smallImageKey = "search";
-					presenceData.smallImageText = "Searching...";
-				} else if (document.location.pathname.includes("/templates")) {
-					presenceData.largeImageKey = "discordtemplates_logo";
-					presenceData.details = "Viewing home page";
-				}
+			delete presenceData.smallImageKey;
+			delete presenceData.smallImageText;
 		}
+	} else if (pathname.includes("/templates/user")) {
+		const userWebsite = document
+			.querySelector("p.mx-auto")
+			?.querySelector("a")
+			?.getAttribute("href");
+		presenceData.details = document.querySelector("h1.text-3xl")?.textContent;
+		presenceData.state = document.querySelector("p.mx-auto")?.textContent;
+		presenceData.largeImageKey =
+			document.querySelector("figure.mx-auto")?.querySelector("img")?.src ??
+			Assets.Logo;
+		presenceData.smallImageKey = Assets.Logo;
+		presenceData.smallImageText = `Discords Templates - Total of ${
+			document
+				.querySelector("section.mt-4")
+				?.querySelector("div.space-x-2")
+				?.querySelectorAll("span")[1]?.textContent
+		}`;
+		presenceData.buttons = [
+			{
+				label: "View User",
+				url: href,
+			},
+		];
+
+		if (userWebsite) {
+			presenceData.buttons.push({
+				label: "View Website",
+				url: userWebsite,
+			});
```

Comment:
Can you give me an example(s) with what this button would link to? Seems like advertising to me

Reply:
Some users have personal websites linked. Will remove.